### PR TITLE
Update creature.md

### PR DESCRIPTION
**Pull request by Gultask**
### Description

- https://github.com/azerothcore/azerothcore-wotlk/pull/17187
- 

### Related Issue

Closes

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

### CODE REVIEW

File: docs/creature.md [hunk 1]
```diff
@@ -55,6 +58,9 @@ Contains individual creature spawn data for each individual spawn of each indivi
 [21]: #npcflag
 [22]: #unitflags
 [23]: #dynamicflags
+[24]: #scriptname
```
Comment (Kitzunu) [inline docs/creature.md:61]: add description for scriptname

Comment (Gultask) [reply docs/creature.md:61]: no

Comment (Gultask) [reply docs/creature.md:61]: I'll also add the Comment column in this PR

Comment (Gultask) [reply docs/creature.md:61]: GH doesn't ping me on reviews fml

**Pull request merged**